client/my-sites/checkout/composite-checkout/lib/translate-payment-method-names.ts

```diff
@@ -54,9 +56,11 @@ export function translateCheckoutPaymentMethodToWpcomPaymentMethod(
 ): WPCOMPaymentMethod | null {
 	// existing cards have unique paymentMethodIds
 	if ( paymentMethod.startsWith( 'existingCard' ) ) {
-		paymentMethod = 'card';
+		paymentMethod = 'existingCard';
```

Comment:
This caused a regression wherein stored cards are filtered out of the list of payment methods available in the "change payment method" form. It should be fixed by D72552-code which explicitly adds stored cards to the list of available payment methods returned by the server.